src/test/runTest.ts

```diff
@@ -10,8 +10,10 @@ async function main() {
 	const tmpHomeDir: string = fs.mkdtempSync("/tmp/vscode-tests");
 	try {
 		const settings = {
+			// eslint-disable-next-line @typescript-eslint/naming-convention
 			"inmanta.ls.enabled": true,
-			"python.defaultInterpreterPath": process.env.INMANTA_EXTENSION_TEST_ENV
+			// eslint-disable-next-line @typescript-eslint/naming-convention
+			"python.defaultInterpreterPath": process.env.INMANTA_EXTENSION_TEST_ENV // eslint-disable-next-line camelcase
```

Comment:
the warning we get is: `Object Literal Property name `inmanta.ls.enabled` must match one of the following formats: camelCase`
I think dots are not allowed

Reply:
But that's horrible because this is a plain string, this could be literally anything, couldn't it? Or does it somehow recognize the context we're in?

Reply:
I don't like it neither. At first I didn't want to add it but I got annoyed of seeing the warning each time when I know nothing is wrong.

Reply:
Looks like JS doesn't really have a separate map type, just objects. Therefore the rules seem to be muddled a bit, because generally (afaik) objects are used a lot like in Python, in which case its keys are identifiers, and should in fact match naming conventions. I can see how this would make linting very difficult. I found this related ticket: typescript-eslint/typescript-eslint#1483. I think it contains some workarounds but I didn't dive too deeply myself.